waspc/src/Wasp/Generator/PackageJsonGenerator.hs

```diff
@@ -1,17 +1,71 @@
 module Wasp.Generator.PackageJsonGenerator
   ( resolveNpmDeps,
+    resolveDependencies,
+    DependencyConflictError (DependencyConflictError),
```

Comment:
It's internal for now, but I don't think it will remain internal due to a further refactoring I'm doing that follows up on this. In fact I think `resolveNpmDeps` in its current form will disappear. Instead there will be a function that can turn `DependencyConflictError` into loggable text. Would that be an implementation of Show or would you recommend creating a custom function for that?

So I'm not going to make this change now, but should I want to create an Internal in the future, can I do so in the same file with another 'module' statement or do I need to create another module?

Reply:
Ok, in that case don't bother with creating additional Internal module.
Yes, it would have to be another file -> you can't create two modules in one Haskell file.

Hmmmm `Show` or not -> I also wonder about that sometimes. I usually end up not using `Show`, as it often feels too generic. Let's put it this way: if there are multiple reasonable ways to show something, then it is best to make them separate functions. If there is a "canonical" way to display something, or it at least feels somewhat "canonically", then go with show.

Here -> hard to say, if I had to make choice I would do standalone function though.